test/driver/exception_handling/run.t

```diff
@@ -44,21 +44,42 @@ caught, so no AST is produced.
 
   $ echo "let x = 1+1. " > impl.ml
   $ echo "let _ = [%gen_raise_located_error]" >> impl.ml
+  $ echo "let _ = [%gen2_raise_located_error]" >> impl.ml
   $ export OCAML_ERROR_STYLE=short
+
+ when the -embed-errors flag is not passed  
   $ ./extender.exe impl.ml
   File "impl.ml", line 2, characters 8-34:
   Error: A raised located error
   [1]
 
+
+ when the -embed-errors flag is  passed
+  $ ./extender.exe -embed-errors impl.ml
+  [%%ocaml.error "A raised located error"]
```

Comment:
We can see that it's not working yet (likely only due to the value of `embed_errors`, which is always `false` in your implementation).

If it were working, both errors("A raised located error" and "A second raised located errors" would be seen)